src/test/java/cz/codecamp/logger/loggers/PrintStreamLoggerTest.java

```diff
@@ -0,0 +1,51 @@
+package cz.codecamp.logger.loggers;
+
+import cz.codecamp.logger.LogLevelEnum;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.PrintStream;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Created by Martin on 15.10.2016.
+ */
+public class PrintStreamLoggerTest {
+
+    private PrintStreamLogger logger;
+    private final ByteArrayOutputStream outContent = new ByteArrayOutputStream();
+    private final ByteArrayOutputStream errContent = new ByteArrayOutputStream();
+
+    @Before
+    public void setUp() throws Exception {
+        PrintStream printStream = new PrintStream(new File("abc.txt"));
+        this.logger = new PrintStreamLogger(printStream);
+        System.setOut(new PrintStream(outContent));
+    }
+
+    @Test
+    public void testMetodyLog(){
+        logger.log(LogLevelEnum.INFO, "Test Message", 2);
+        boolean actual = outContent.toString().contains("Zalogovano");
```

Comment:
Tohle fakt ne... Musíš otestovat, že to zapisuje do toho souboru, co vytvoříš. Ne že to něco vypíše na stdout :)